### ISSUE TITLE
./rguard -0 does not work

### ISSUE THREAD

**Issue by prietus** [ryran/reboot-guard#5]
it does not handle (rm_dir) /run/systemd/system/{halt,shutdown,poweroff}.target.d/{}.conf files

tested in archlinux with systemd 233-3

```
rguard -1 -d debug output:
DEBUG: Adding RefuseManualStart=yes to /run/systemd/system/poweroff.target.d/reboot-guard.conf
WARNING: ☹  Blocked poweroff.target
DEBUG: Adding RefuseManualStart=yes to /run/systemd/system/reboot.target.d/reboot-guard.conf
WARNING: ☹  Blocked reboot.target
DEBUG: Adding RefuseManualStart=yes to /run/systemd/system/halt.target.d/reboot-guard.conf
WARNING: ☹  Blocked halt.target
DEBUG: Reloading systemd via: systemctl daemon-reload
INFO: Exiting due to -1 or -0 option
```

rguard -0 -d debug output:

```
DEBUG: Unit poweroff.target already configured with RefuseManualStart=no
DEBUG: Unit reboot.target already configured with RefuseManualStart=no
DEBUG: Unit halt.target already configured with RefuseManualStart=no
```

**Comment by ryran** [comment]
The latest systemd version I've tested on is Fedora25's 231.

Based on your output, it looks like `rguard -1` is not actually working even though it reports no errors. I don't have any bright ideas here. As you know, the locking operation writes to `{poweroff,reboot,halt}.target.d/reboot-guard.conf` files and then runs `systemctl daemon-reload` to apply the changes ... the unblocking operation checks those units with `systemctl show (poweroff|reboot|halt).target -p RefuseManualStart` commands and if the command succeeds AND reports `no`, then you get the `already configured with RefuseManualStart=no` message that you're seeing.

Maybe in this this new version they've made it so that `daemon-reload` somehow doesn't apply changes to the `RefuseManualStart` property... show me:

```
rguard -1 -v debug
cat /run/systemd/system/poweroff.target.d/reboot-guard.conf
systemctl show poweroff.target -p RefuseManualStart
```

**Comment by prietus** [comment]
well, -1 works (I cannot reboot/poweroff/halt), it's just -0 does not. it's like it does not handle the removal of target.d/ dirs.
ill check what you said.

**Comment by prietus** [comment]
ah, it works, it's just the python shebang should be set to python2 since archlinux defaults to python3.
sorry for the noise and thanks for the script.

**Comment by ryran** [comment]
Ah! Good catch @prietus ... see: #6 
I've published a new release with that change.

EDIT: PS: I'd love to hear how and why you or anyone you know uses reboot-guard! If you have a sec, feel free to add a new issue with the `feedback-stories` label.
